interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
 
 ### DataFactory
 
 **Methods:**
 
-- `.iri(String iri)` returns a new instance of IRI.
-- `.blankNode()` returns a new instance of BlankNode.
+- `.sym(String iri)` returns a new instance of IRI.
+- `.bNode()` returns a new instance of BlankNode.
```

Comment:
@RubenVerborgh RDF Interfaces merely provides a primitive for representing RDF structures in memory; for queries it serves the same purpose that Array or String does to an asynchronous RDB query library (think SQL or IndexedDB): Representing result sets.

There's other interfaces that might be better targets for asynchronicity, e.g. [RDF API](http://www.w3.org/TR/rdf-api/). Similarly, my adaption of IndexedDB to RDF graphs, that I call [IndexedGraph](https://lists.w3.org/Archives/Public/public-rdfjs/2013Oct/0040.html), still uses RDF Interfaces for handing values in memory.

If for this purpose, the existing implementation is insufficient or has problems (and it does, hence the desire to take up work), we should be at a point where we understand and can clearly articulate why.